panic error when using azcopy benchmark (used to work fine till today)

### Which version of the AzCopy was used? 
##### Note: The version is visible when running AzCopy without any argument
tried with 10.10 and 10.9
### Which platform are you using? (ex: Windows, Mac, Linux)
Windows 10 (all the updates)
### What command did you run?
##### Note: Please remove the SAS to avoid exposing your credentials. If you cannot remember the exact command, please retrieve it from the beginning of the log file.
azcopy bench https://hbstotest.file.core.windows.net/hamzasto?xxxxxmySASKeyxxxxx" --file-count 100 --size-per-file 1G
### What problem was encountered?
got this panic error : 

panic: couldn't create job part plan file "b73ea971-b6cd-3f48-5e4f-39823d48a92b--00000.steV16": open C:\Users\adm-id184826\.azcopy/plans/b73ea971-b6cd-3f48-5e4f-39823d48a92b--00000.steV16: L’opération demandée n’a pu s’accomplir sur un fichier ayant une section mappée utilisateur ouverte.

goroutine 991 [running]:
github.com/Azure/azure-storage-azcopy/ste.JobPartPlanFileName.Create(0xc015fbfc40, 0x32, 0xb73ea97100000000, 0x82394f5e3f48b6cd, 0x2ba9483d, 0x400000000000001, 0x3, 0x0, 0x0, 0x0, ...)
        /home/vsts/work/1/s/ste/JobPartPlanFileName.go:134 +0x276c
github.com/Azure/azure-storage-azcopy/ste.ExecuteNewCopyJobPartOrder(0xb73ea97100000000, 0x82394f5e3f48b6cd, 0x2ba9483d, 0x400000000000001, 0x3, 0x0, 0x0, 0x0, 0xc000260420, 0x60, ...)
        /home/vsts/work/1/s/ste/init.go:143 +0xb9
github.com/Azure/azure-storage-azcopy/cmd.inprocSend(0xeebe45, 0x10, 0xd45080, 0xc097b65340, 0xd450c0, 0xc097e527e0, 0xe40701, 0xc097e527e0)
        /home/vsts/work/1/s/cmd/rpc.go:44 +0x91f
github.com/Azure/azure-storage-azcopy/cmd.glob..func4(0xeebe45, 0x10, 0xd45080, 0xc097b65340, 0xd450c0, 0xc097e527e0)
        /home/vsts/work/1/s/cmd/rpc.go:36 +0x65
github.com/Azure/azure-storage-azcopy/cmd.(*copyTransferProcessor).sendPartToSte(0xc097e80120, 0x500000000, 0x1031740, 0xc0205e0a00)
        /home/vsts/work/1/s/cmd/zc_processor.go:132 +0x8d
github.com/Azure/azure-storage-azcopy/cmd.(*copyTransferProcessor).dispatchFinalPart(0xc097e80120, 0xc097e80120, 0xf155fd, 0x160ef40)
        /home/vsts/work/1/s/cmd/zc_processor.go:108 +0x45
github.com/Azure/azure-storage-azcopy/cmd.newRemoveEnumerator.func2(0xc0205e0a00, 0x0)
        /home/vsts/work/1/s/cmd/removeEnumerator.go:77 +0x3e
github.com/Azure/azure-storage-azcopy/cmd.(*copyEnumerator).enumerate(0xc0205e0a40, 0xc0205e0a40, 0x0)
        /home/vsts/work/1/s/cmd/zc_enumerator.go:673 +0x9f
github.com/Azure/azure-storage-azcopy/cmd.(*cookedCopyCmdArgs).processCopyJobPartOrders(0xc000081800, 0x0, 0x0)
        /home/vsts/work/1/s/cmd/copy.go:1266 +0x865
github.com/Azure/azure-storage-azcopy/cmd.(*cookedCopyCmdArgs).process(0xc000081800, 0xc0163fbfc0, 0xc2961f)
        /home/vsts/work/1/s/cmd/copy.go:1045 +0xbc
github.com/Azure/azure-storage-azcopy/cmd.(*cookedCopyCmdArgs).launchFollowup.func1(0xc097b5c900, 0xc000080c00)
        /home/vsts/work/1/s/cmd/copy.go:1352 +0x79
created by github.com/Azure/azure-storage-azcopy/cmd.(*cookedCopyCmdArgs).launchFollowup
        /home/vsts/work/1/s/cmd/copy.go:1349 +0x66

### How can we reproduce the problem in the simplest way?
ran the same commande from 3 differents computers (which used to work fine) , the panic error appears randomly , sometimes at the end of the copy sometime in the middle (stopping the bench) . 
### Have you found a mitigation/solution?

no

--- a/cmd/benchmark.go
+++ b/cmd/benchmark.go
@@ -241,6 +241,7 @@ func (raw rawBenchmarkCmdArgs) createCleanupJobArgs(benchmarkDest common.Resourc
 	rc.setMandatoryDefaults()
 
 	cooked, err := rc.cook()
+	cooked.jobID = common.NewJobID() // Override the job ID that cook gave us-- That would cause us to fail deletion.
 	cooked.isCleanupJob = true
 	cooked.cleanupJobMessage = "Running cleanup job to delete files created during benchmarking"
 	return &cooked, err

--- a/ste/JobPartPlanFileName.go
+++ b/ste/JobPartPlanFileName.go
@@ -17,6 +17,11 @@ import (
 
 type JobPartPlanFileName string
 
+func (jppfn *JobPartPlanFileName) Exists() bool {
+	_, err := os.Stat(jppfn.GetJobPartPlanPath())
+	return err == nil
+}
+
 func (jppfn *JobPartPlanFileName) GetJobPartPlanPath() string {
 	return fmt.Sprintf("%s%s%s", JobsAdmin.AppPathFolder(), common.AZCOPY_PATH_SEPARATOR_STRING, string(*jppfn))
 }
@@ -70,6 +75,10 @@ func (jpfn JobPartPlanFileName) Map() *JobPartPlanMMF {
 
 // createJobPartPlanFile creates the memory map JobPartPlanHeader using the given JobPartOrder and JobPartPlanBlobData
 func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest) {
+	if jpfn.Exists() {
+		return // This is *probably* for a cleanup job.
+	}
+
 	// Validate that the passed-in strings can fit in their respective fields
 	if len(order.SourceRoot.Value) > len(JobPartPlanHeader{}.SourceRoot) {
 		panic(fmt.Errorf("source root string is too large: %q", order.SourceRoot))

--- a/ste/JobPartPlanFileName.go
+++ b/ste/JobPartPlanFileName.go
@@ -76,7 +76,7 @@ func (jpfn JobPartPlanFileName) Map() *JobPartPlanMMF {
 // createJobPartPlanFile creates the memory map JobPartPlanHeader using the given JobPartOrder and JobPartPlanBlobData
 func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest) {
 	if jpfn.Exists() {
-		panic(fmt.Sprintf("Duplicate job created. You probably shouldn't ever see this, but if you do, try cleaning out", JobsAdmin.(*jobsAdmin).planDir))
+		panic(fmt.Sprint("Duplicate job created. You probably shouldn't ever see this, but if you do, try cleaning out", JobsAdmin.(*jobsAdmin).planDir))
 	}
 
 	// Validate that the passed-in strings can fit in their respective fields

--- a/ste/JobPartPlanFileName.go
+++ b/ste/JobPartPlanFileName.go
@@ -76,7 +76,7 @@ func (jpfn JobPartPlanFileName) Map() *JobPartPlanMMF {
 // createJobPartPlanFile creates the memory map JobPartPlanHeader using the given JobPartOrder and JobPartPlanBlobData
 func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest) {
 	if jpfn.Exists() {
-		return // This is *probably* for a cleanup job.
+		panic(fmt.Sprintf("Duplicate job created. You probably shouldn't ever see this, but if you do, try cleaning out", JobsAdmin.(*jobsAdmin).planDir))
 	}
 
 	// Validate that the passed-in strings can fit in their respective fields

